crates/compiler/builtins/bitcode/src/dec.zig

```diff
@@ -887,15 +887,15 @@ test "toStr: -0.45" {
 }
 
 test "toStr: 0.00045" {
-    var dec: RocDec = .{ .num = 000450000000000000 };
+    var dec: RocDec = .{ .num = 450000000000000 };
```

Comment:
Latest Zig won't allow leading zeros in integer literals. A bit annoying in this case since this integer is really a fixed-point number, and the leading zeros were helpful for understanding where the first significant digit was.

Reply:
For those interested, a small discussion on why this was merged:
https://github.com/ziglang/zig/pull/12443